### ISSUE TITLE
QueueMessage DequeueCount is still str in 0.30.x

### ISSUE THREAD

**Issue by bearrito** [Azure/azure-storage-python#147]
Referencing prior issues

1: https://github.com/Azure/azure-storage-python/issues/57

2: https://github.com/Azure/azure-storage-python/issues/16

The dequeue_count on  is still a string in the QueueMessage, other fields have been promoted to better types. 

Breaks code such.

```

 if message.dequeue_count > 3:
     print "Poison Message"
     self.queue_service.delete_message(self.config.queue_name, message.id, message.pop_receipt)

```

**Comment by emgerner-msft** [comment]
I documented this as an int but it looks like I didn't parse it correctly in the deserialization code. Will fix and return it as an int in our next release. Thanks for submitting this!

**Comment by emgerner-msft** [comment]
Fixed as of 0.30.1.
